electricitylci/main.py

```diff
@@ -132,4 +144,4 @@ def main():
         config.model_specs=config.build_model_class(args.model_config)
     else:
         config.model_specs=None
-    main()
+        main()
```

Comment:
I think you still need to run main() if an argument is passed, hence why it was outside the else statement

Reply:
This explains why nothing happen when I tried to specify a configuration from the command line.

Reply:
Sorry about this. I was returning some variables and mistakenly added a space.